understanding/22/dragging-movements.html

```diff
@@ -37,19 +37,21 @@ <h2>Intent of Dragging Movements</h2>
+     <p>Success Criteria 2.1.1 Keyboard and 2.1.3 Keyboard (No Exception) require dragging features to be keyboard accessible, however, it is possible to create an interface that works with dragging and keyboard controls that does not work using only clicks or taps.</p>
 
-      <p>2.1.1 Keyboard and 2.1.3 Keyboard (No Exception) require dragging features to be keyboard accessible, however, it is possible to create an interface that works with dragging and keyboard controls that does not work using only clicks or taps.</p>
+    <p>This Success Criterion applies to dragging motions as opposed to pointer gestures which are covered in Success Criterion 2.5.1 Pointer Gestures. Pointer gestures include directional path-based as well as multi-point gestures. In contrast, for dragging motions, only the start and end point of the motion matters, not the actual path.</p>
 
       <p>There may be cases where a dragging movement is essential, for example, drawing a freeform figure, or playing a game where the aim is to follow a particular path as accurately as possible. The criterion has an exception for these cases where the task requires dragging.</p>
 
       <p>Some tasks may typically involve dragging, such as sorting items in a list, but as there are other ways to achieve that task they are not considered essential.</p>
       
       <h3>Distinguishing dragging movements from path-based pointer gestures</h3>
-      <p>A dragging movement is a pointer interaction where only the endpoints matter. Once the pointer engages with a target, the target (or a position mark related to the target) will follow the pointer regardless of the direction of the pointer movement. The target's movement may be constrained to one axis and often, to a particular range on that axis, but within those constraints, the x or y position of the target will mirror the corresponding x or y position of the pointer until the pointer disengages the target.</p> 
-      <p>In contrast to dragging movements, path-based gestures involve at least an initial directionality of the pointer movement. This is defined by stating that the path has to go through at least one intermediate point to qualify as pointer gesture. The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined. For more details, refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">Understanding Success Criterion 2.5.1 Pointer Gestures</a>.</p>
+      
+      <p>Dragging movements covered in this Success Criterion are pointer interactions where only the endpoints matter. Once the pointer engages with a target, the direction of the pointer motion does not factor into the interaction until the pointer disengages that target. Dragging is a motion between the start and end points without an intermediate point. The motion can go in any direction and is entirely undefined. </p> 
```

Comment:
This paragraph seems a bit redundant, there two similar statements about dragging motions (we also need tosettle for eirher ‚movement‘ or ‚motion‘)

Reply:
See revision for reducing redundancy.

Regarding dragging movement versus draggin motion, a dragging movement seems like what is defined and provided by the system and a dragging motion is what is performed by the user. Does that make sense? I can take a pass through the whole document and update to meet these definitions.